### PR TITLE
feat(core): Upgrade chokidar, braces, and micromatch to address CVE-2024-4067 & CVE-2024-4068 (no-changelog)

### PR DESCRIPTION
**Pull request by netroy**
- [CVE-2024-4067](https://github.com/advisories/GHSA-952p-6rrq-rcjv)
- [CVE-2024-4068](https://github.com/advisories/GHSA-grv7-fg5c-xmjg)

## Review / Merge checklist

- [x] PR title and summary are descriptive

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
     ],
     "overrides": {
       "@types/node": "^18.16.16",
-      "chokidar": "3.5.2",
+      "chokidar": "^4.0.1",
```
Comment (valya) [inline package.json:72]: Can we move this to `catalog:`?

Comment (netroy) [reply package.json:72]: this needs to be in `pnpm.overrides` because some dev dependencies are pulling on older versions, even though they work with 4.x

Comment (valya) [reply package.json:72]: Can we move the other two uses then?

Comment (netroy) [reply package.json:72]: we unfortunately can't remove it from the nodes package, but I've move the version to catalog. 
but, for now at least, we need to keep the override as well.

**Review by valya — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.63.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.63.0)

**Pull request merged**